### PR TITLE
Update QoS-in-Teams-clients.md

### PR DESCRIPTION
**Pull request by baardhermansen**
Added table name to first table and renumbered the second one. The article text seems to have been a cut and paste from another article, hence the numbering was out of order.
Changed list numbering to follow Microsoft markdown standards.
Corrected codeblock markdown.
Rewrote a couple of sentences to mimic "write like you speak".
Removed the image of command prompt.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.
--

File: Teams/QoS-in-Teams-clients.md [hunk 1]
```diff
@@ -21,83 +21,83 @@ appliesto:
...
+1. Under the heading **Specify the source port number**, select **From this source port or range**. In the accompanying text box, type the port range reserved for audio transmissions. For example, if you reserved ports 50000 through ports 50019 for audio traffic, enter the port range using this format: **50000:50019**. Click **Finish**.
 
-11. Repeat steps 5-10 to create policies for Video and Application/Desktop Sharing, substituting the appropriate values in steps 6 and 10.
+1. Repeat steps 5-10 to create policies for Video and Application/Desktop Sharing, substituting the appropriate values in steps 6 and 10.
 
-The new policies you’ve created won’t take effect until Group Policy has been refreshed on your client computers. Although Group Policy periodically refreshes on its own, you can force an immediate refresh by following these steps:
+The new policies you’ve created won’t won't ’ ' ' ’ take effect until Group Policy has been refreshed on your client computers. Although Group Policy periodically refreshes on its own, you can force an immediate refresh by following these steps:
```
Comment (JohanFreelancer9) [inline Teams/QoS-in-Teams-clients.md]:
```suggestion
The new policies you’ve created won’t take effect until Group Policy has been refreshed on your client computers. Although Group Policy periodically refreshes on its own, you can force an immediate refresh by following these steps:
```

**Comment by baardhermansen** [comment]
> Copy Review

Review completed.

**Comment by LolaJacobsen** [comment]
@siunies please review these proposed changes.

**Comment by LolaJacobsen** [comment]
@siunies - see the 1:1 chat where I pointed out the sections that need your review.

**Pull request merged**